vkcs/data_source_vkcs_blockstorage_snapshot_test.go

```diff
@@ -30,7 +30,6 @@ func TestAccBlockStorageSnapshotDataSource_basic(t *testing.T) {
 	}
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheckBlockStorage(t) },
```

Comment:
what's the reason to remove prechecks?

Reply:
We try to eliminate usage of env variables for automation purposes, thus these checks are no longer needed